src/main/java/com/org/trustservice/excpetion/DatabaseException.java

```diff
@@ -0,0 +1,17 @@
+package com.org.trustservice.excpetion;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class DatabaseException extends RuntimeException {
```

Comment:
Database exception sounds like a severe exception, at least by the name. So ideally wrap it in IOException or something. Runtime exceptions are for code specific stuffs.